logbook-api/pom.xml

```diff
@@ -18,7 +18,7 @@
         <dependency>
             <groupId>org.organicdesign</groupId>
             <artifactId>Paguro</artifactId>
-            <version>3.1.4</version>
+            <version>3.10.3</version>
```

Comment:
Any version higher than `3.1.4` is compiled with java 11.

Reply:
Java compiler version was updated to 17 (#1434), so the tests pass now.